app/views/website_members/index.html.haml

```diff
@@ -18,9 +19,12 @@
             %th.text-right Actions
         %tbody.websites
           - @website_members.each do |website_member|
-            %tr
-              %td= website_member.user.name
-              %td= website_member.user.email
-              %td= website_member.role
-              %td.text-right
-                = link_to ' Delete', "/website_members/#{website_member.id}", :method => :delete, data: {confirm: "Are you sure?"}, :class => "delete_website btn btn-danger pull-right fa fa-close"
\ No newline at end of file
+            - unless @current_website_member == website_member
```

Comment:
let's not do this stupid thing here and just not select the current user by adjusting the query in the controller